src/nylas-connection.js

```diff
@@ -147,7 +147,7 @@ module.exports = class NylasConnection {
 
         if (error || response.statusCode > 299) {
           if (!error) {
-            error = new Error(body.message);
+            error = new Error(JSON.stringify(body));
```

Comment:
hmm. I do definitely want to capture the rest of the data in this error message but I want to format it nicely. Maybe for now can we just add another `if` clause below to format this particular edge case, and let's revisit Error handling later?